src/rgw/rgw_rados.cc

```diff
@@ -174,6 +176,18 @@ int rgw_init_ioctx(librados::Rados *rados, const rgw_pool& pool, IoCtx& ioctx, b
     if (r < 0 && r != -EEXIST) {
       return r;
     }
+    if (mostly_omap) {
+      // set pg_autoscale_bias
+      bufferlist inbl;
+      int r = rados->mon_command(
+	"{\"prefix\": \"osd pool set\", \"pool\": \"" +
+	pool.name + "\", \"var\": \"pg_autoscale_bias\": \"4.0\"}",
```

Comment:
maybe we could lift the 4.0 value out to a constant (not a #define)?

Reply:
updated!  added options for both pg_num_min and autoscale_bias